resources/sass/components/_form.scss

```diff
@@ -0,0 +1,16 @@
+// form
+
+/**
+ * Required field
+ */
+.form-required:after {
+  content: '';
+  vertical-align: super;
+  display: inline-block;
+  background-image: url('../../images/required.svg');
```

Comment:
are we guessing ? can you please verify before submitting.

Reply:
We did, no guessing we checked.